### PR TITLE
New Map for Downloading

### PR DESCRIPTION
**Pull request by CrazyGerman**

### CODE REVIEW

**Comment by ron-murhammer** [comment]
@CrazyGerman So the changes to this file look good now. But it looks like your map repo structure needs adjusted: https://github.com/triplea-maps/house_of_habsburg

The folder structure needs to be similar to other map repos in that you need to have everything in a folder called "map" (you can ignore readme and gitignore): https://github.com/triplea-maps/the_pact_of_steel

**Comment by ron-murhammer** [comment]
@CrazyGerman The one thing you are missing is a small image of the map to add to the description as well. Most add this to their map repo and then we can just link to it from here.

**Comment by CrazyGerman** [comment]
Thanks for your specific responses. I couldn't find information about how to do the YAML portion anywhere.

The map fix is easy, I will update it later.

I can't attempt to put the small image in as I'm at work right now. How do I path that link? Just copy and paste from other maps and change the name to house_of_habsburg?

**Comment by ron-murhammer** [comment]
@CrazyGerman Yeah, the yaml portion is mostly just going off the others as examples. For the image, if you create a folder at the same level as 'map' called 'description' then the URL will be something like: https://raw.githubusercontent.com/triplea-maps/house_of_habsburg/master/description/house_of_habsburg.png

File: triplea_maps.yaml [hunk 1]
```diff
@@ -1815,6 +1815,15 @@
     <br>
     Ultimate World Variants
     <br>Includes several Ultimate World Variants including FFAs
+- mapName: House of Habsburg
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/house_of_habsburg/archive/master.zip
+  version: 1
+  img:
```
Comment (RoiEXLab) [inline triplea_maps.yaml]: The travis build failed because no URL has been specified here.
Either you add a valid URL here, or drop this line entirely.

Comment (DanVanAtta) [reply triplea_maps.yaml]: @CrazyGerman you can also post an image to a github issue or any github dialog, it'll give you a URL; you can use that URL here too if it's easy than checking in to the map repository.

**Pull request merged**